README.md

```diff
@@ -66,7 +62,7 @@ axe.run(function (err, results) {
 
 ## Supported Browsers
 
-The [axe API](doc/API.md) fully supports the following browsers:
+The [aXe API](doc/API.md) fully supports the following browsers:
```

Comment:
I thought we were getting rid of the capital X? cc @caitlinthefirst

Reply:
We are. I clearly missed some of them.

Reply:
@caitlinthefirst whatever you prefer! You're a member of the project, so if you'e comfortable you should be able to check out the `marketing-readme` branch and add a new commit. Otherwise I'm happy to make a change for you!

Reply:
I've fixed them.